src/Symfony/Component/Finder/Tests/Iterator/SortableIteratorTest.php

```diff
@@ -33,7 +33,7 @@ public function testAccept($mode, $expected)
         if (!is_callable($mode)) {
             switch ($mode) {
                 case SortableIterator::SORT_BY_ACCESSED_TIME :
-                    file_get_contents(self::toAbsolute('.git'));
+                    file_get_contents(self::toAbsolute('.foo/bar'));
```

Comment:
not really necessary, now that's is skipped, but was failing on windows because .git is a directory.